packages/api/test/e2e/promise-alex-archive.spec.ts

```diff
@@ -34,25 +37,23 @@ describe.skip('alex archive queries (local)', () => {
 
   // https://github.com/polkadot-js/api/issues/846
   it('handles toJSON with no issues', async (done) => {
-    const signedBlock = await api.rpc.chain.getBlock('0x85c62b581f38cb81c3e443d34392672beb1fb877017fd7237cc87704113259dc');
-    const failed = signedBlock.block.extrinsics.filter((extrinsic) => {
-      try {
-        const json = extrinsic.method.toJSON();
-
-        console.error(json);
-
-        return false;
-      } catch (error) {
-        console.log(extrinsic.method);
-        console.log(extrinsic.method.keys());
-        console.error(error);
-
-        return true;
-      }
-    });
-
-    expect(failed).toHaveLength(0);
-
-    done();
+    api.rpc.chain.getBlock('0x6f6f9bba0eed8e3ae9446c37eee763f93118b52a315a7b46090453ba6288da1f', (result: SignedBlock) => {
```

Comment:
I'm not sure we should change the block hash

Reply:
reverted.